modules/core/shared/src/main/scala/scaladex/core/model/Artifact.scala

```diff
@@ -30,9 +29,13 @@ case class Artifact(
     isNonStandardLib: Boolean,
     platform: Platform,
     language: Language,
-    fullScalaVersion: Option[SemanticVersion]
+    fullScalaVersion: Option[SemanticVersion],
+    scaladocUrl: Option[Url],
+    versionScheme: Option[String],
+    developers: Option[List[Contributor]]
```

Comment:
Maybe for `developers` we should have a `Seq[Contributor]` rather than a `Option[Seq[Contributor]]` to avoid the double level of nesting.

```suggestion
    developers: Seq[Contributor]
```